projects/ngx-launcher/src/lib/model/dependency-check.model.ts

```diff
@@ -4,4 +4,5 @@ export class DependencyCheck {
   projectName: string;
   projectVersion: string;
   spacePath: string;
+  targetEnvironment: string;
```

Comment:
```targetEnviroment``` is used by the only standalone launcher. So, can you make as an optional attribute?